rep/processors/CtGovPdfProcessor.py

```diff
@@ -35,7 +46,9 @@ def _process_blob_wrapper(self, blob):
             t2 = "".join(re.findall(self.UNKOWN_PATTERN, votes[i][1])).strip()
             vote_list.append((t1, t2))
 
-        self._write_to_csv(
+        self.mark_processed(sourceUrl)
```

Comment:
this should be called after `CtGovProcessor#_write_to_db`. If the database write fails, then we want to try processing this again later, but if you mark it as processed first, it won't get processed again on the next run, even if the write fails

Reply:
I'll have to think about this, because right now the way we're "handling" PDFs that aren't actual votes is just by catching an exception. But I can definitely make it more robust, so that it only marks things as processed when it either finishes, or catches the exception